### DISCUSSION TITLE
Quote a tweet

### DISCUSSION THREAD

**Discussion by Bingshamoka** [noweh/twitter-api-v2-php#22] [Ideas]
How can we quote a tweet?

**Comment by noweh** [comment]
Quote a tweet? Like a Retweet? Thanks for the precision.

**Reply by noweh** [reply]
After checking, it seems to be something like this, with "quote_tweet_id" containing the identifier:


    use Noweh\TwitterApi\Client;

    $settings = [
        'account_id' => 1234567,
        'consumer_key' => 'CONSUMER_KEY',
        'consumer_secret' => 'CONSUMER_SECRET',
        'bearer_token' => 'BEARER_TOKEN',
        'access_token' => 'ACCESS_TOKEN',
        'access_token_secret' => 'ACCESS_TOKEN_SECRET'
    ];

    $client = new Client($settings);
    $return = $client->tweet()->performRequest('POST', ['text' => 'This is a test....', 'quote_tweet_id' => 'XXX']);
